torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19949,6 +19949,16 @@ def __init__(
     ElementwiseUnaryPythonRefInfo(
         "_refs.abs",
         torch_opinfo_name="abs",
+        skips=(
+            # Reference result was farther (0.0) from the precise computation
+            # than the torch result was (nan)!
```

Comment:
interesting, do you know what inputs result in 0 vs nan?

Reply:
Haven't checked. Will try to check them tomorrow and update here.